src/PosthogAnalytics.ts

```diff
@@ -200,16 +205,17 @@ export class PosthogAnalytics {
         };
     }
 
-    private capture(eventName: string, properties: posthog.Properties) {
+    private capture(eventName: string, properties: posthog.Properties, options?: IPostHogEventOptions) {
         if (!this.enabled) {
             return;
         }
         const { origin, hash, pathname } = window.location;
         properties["redactedCurrentUrl"] = getRedactedCurrentLocation(origin, hash, pathname);
-        this.posthog.capture(eventName, {
-            ...this.propertiesForNextEvent,
-            ...properties,
-        });
+        this.posthog.capture(
+            eventName,
+            { ...this.propertiesForNextEvent, ...properties },
+            options as any, // No proper type definition in the posthog library
```

Comment:
We can't assert they'll add the timestamp field into options, they could request changes